src/jit/CMakeLists.txt

```diff
@@ -130,6 +130,7 @@ if (WIN32)
     inline.h
     inlinepolicy.h
     instr.h
+    instrxarch.h
```

Comment:
instrxarch.h _and_ instrsxarch.h -- that is going to be confusing.

Reply:
This is one of the reasons I had renamed `instrsxarch.h` t `instrslistxarch.h` originally.